packages/riipen-ui-docs/src/pages/components/chip/Hover.jsx

```diff
@@ -28,6 +28,26 @@ export default function Hover() {
         <Chip label="Warning" hover color="warning" />
         <span style={style} />
         <Chip label="Negative" hover color="negative" />
+        <span style={style} />
```

Comment:
This file already existed, so I'm going to assume hover support was supposed to exist, but maybe got forgotten. :)